packages/api/src/external/ehr/athenahealth/command/get-patient.ts

```diff
@@ -39,17 +46,21 @@ export async function getPatientIdOrFail({
   athenaPracticeId,
   athenaPatientId,
   accessToken,
+  api,
   useSearch = false,
   triggerDq = false,
 }: {
   cxId: string;
   athenaPracticeId: string;
   athenaPatientId: string;
   accessToken?: string;
+  api?: AthenaHealthApi;
   useSearch?: boolean;
   triggerDq?: boolean;
 }): Promise<string> {
-  const { log } = out(`AthenaHealth getPatient - cxId ${cxId} athenaPatientId ${athenaPatientId}`);
+  const { log } = out(
+    `AthenaHealth getPatientIdOrFail - cxId ${cxId} athenaPracticeId ${athenaPracticeId} athenaPatientId ${athenaPatientId}`
```

Comment:
nit: this is kinda wordy and might take up too much space on cloud watch logs. maybe shorten to smth like:
```
`athena getPatientIdOrFail - cxId ${cxId} practiceId 
${athenaPracticeId} patientId ${athenaPatientId}`
```

Reply:
I think mapping to the exact variable names is best so that's it's clear what we're referring to. Not sure it'll be too onerous on the logs but will fix if you really want it.